src/lib/holocene/api-pagination.svelte

```diff
@@ -226,7 +226,7 @@
       <Input
         icon="search"
         id="api-pagination-search-input"
-        class="grow"
+        class="w-full sm:w-1/2"
```

Comment:
I think we want to keep this `grow` 
```suggestion
        class="grow"
```

| Before | After |
|-|-|
|![Screenshot 2024-10-07 at 9 53 05 AM](https://github.com/user-attachments/assets/e0f93ac3-cff0-4038-a848-8a6582953f2f)|![Screenshot 2024-10-07 at 10 02 31 AM](https://github.com/user-attachments/assets/e5591530-f517-434c-948e-bff931db0c8e)|

Reply:
Let me know if this seems better! Thank you Laura! <3